pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/migration/CohortTagMigrator.java

```diff
@@ -21,8 +21,9 @@ public CohortTagMigrator(String index, String realm, CohortTagDao cohortTagDao)
     protected Map<String, Object> getDataByRealm() {
         Map<String, List<CohortTag>> cohortTags = cohortTagDao.getCohortTagsByInstanceName(realm);
         int tagsFromRealm = cohortTags.size();
-        AdditionalCohortTagsRetriever.fromRealm(realm)
-                .ifPresent(retriever -> retriever.mergeRecords(cohortTags));
+        // TODO: Remove this via PEPPER-1339. -DC
+        //AdditionalCohortTagsRetriever.fromRealm(realm)
+        //        .ifPresent(retriever -> retriever.mergeRecords(cohortTags));
```

Comment:
Is my understanding correct? The new way of copying Cohort Tags copies the participant's cohort tags for OS1 in both the database and ES for OS2, and so in case of an ES export we don't need any additional checks to bundle os1 and os2 tags together for ES export?

Reply:
The prior code here copied both OS1 and OS2 tags to each index.
I was concerned about removing it because of how the participant list stuff uses cohort tags.
But now that we have better cohort tag requirements and implementation I do not think it is needed.